fdbclient/ServerKnobs.cpp

```diff
@@ -102,7 +102,7 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( PUSH_STATS_SLOW_RATIO,                                 0.5 );
 	init( TLOG_POP_BATCH_SIZE,                                  1000 ); if ( randomize && BUGGIFY ) TLOG_POP_BATCH_SIZE = 10;
 	init( TLOG_POPPED_VER_LAG_THRESHOLD_FOR_TLOGPOP_TRACE,     250e6 );
-	init( ENABLE_DETAILED_TLOG_POP_TRACE,                       true );
+	init( ENABLE_DETAILED_TLOG_POP_TRACE,                      false );
```

Comment:
can we add buggify? 
if ( randomize && BUGGIFY ) ENABLE_DETAILED_TLOG_POP_TRACE = true;

Reply:
otherwise, simulation may not catch bug when the knob is set true.

Reply:
Done.